style.css

```diff
@@ -0,0 +1,54 @@
+header {
+  display: flex;
+  flex-direction: row;
+  padding-top: 5%;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.welcome {
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  text-align: left;
+}
+
+body {
```

Comment:
- [ ] [Optional] This is not required but for best practices, it would be better if you used classes to style elements on the page instead of tag names